koku/api/report/test/test_queries.py

```diff
@@ -222,14 +224,18 @@ def setUp(self):
     def test_init(self):
         """Test that we can instantiate a minimal ReportQueryHandler."""
         params = self.mocked_query_params("", self.mock_view)
-        rqh = create_test_handler(params)
+        with patch("api.report.queries.ReportQueryHandler.query_table", new=PropertyMock) as mock:
+            mock.return_value = GCPCostEntryLineItemDailySummary
+            rqh = create_test_handler(params)
```

Comment:
If you add `self.provider = None` to the `__init__` of `ReportQueryHandler` and then add  `views=MagicMock(),` to the `_mapper = Mock` of the `TestableReportQueryHandler`, you can avoid adding this patch to all of these tests. I don't think it's necessarily good practice to add a GCP table to these tests when these tests are _not_ GCP related.